flow-nodes/api-builder-plugin-logger/src/flow-nodes.yml

```diff
@@ -6,9 +6,11 @@ flow-nodes:
     category: general
     methods:
       log:
+        name: Output Log
         description: Writes to the output log.
         parameters:
           level:
+            name: Loglevel
```

Comment:
Why didn't you use a space?  `Loglevel => Log level`

Reply:
Resolved now.